src/main/java/io/mantisrx/api/push/MantisSSEHandler.java

```diff
@@ -160,20 +156,32 @@ private boolean isSubmitAndConnect(HttpRequest request) {
     }
 
     @Override
-    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
-        if (this.subscription != null && !this.subscription.isUnsubscribed()) {
-            this.subscription.unsubscribe();
-        }
-        cause.printStackTrace();
-        ctx.close();
+    public void channelUnregistered(ChannelHandlerContext ctx) throws Exception {
+        log.info("Channel is unregistered: {}", ctx.channel());
```

Comment:
log `url` in these messages too? i'm not familiar with what logged with `channel`.

Reply:
The channel has the host and port information only. One example:
```
Channel is inactive: [id: 0x3df221ac, L:/0:0:0:0:0:0:0:1:7004 ! R:/0:0:0:0:0:0:0:1:62830]
```
I'll add the URI in logging.

Reply:
After recent commit,
```2023-01-26 10:48:11,281 INFO  io.mantisrx.api.push.MantisSSEHandler [Salamander-ClientToZuulWorker-0] Channel [id: 0xb7e5e05b, L:/0:0:0:0:0:0:0:1:7004 ! R:/0:0:0:0:0:0:0:1:63289] is unregistered. URI: /jobconnectbyname/SharedMrePublishEventSource?clientId=mingliangl-1674712485&subscriptionId=mingliangl-1674712485&targetApp=datameshprocessorsql-laptop-mingliangl&criterion=SELECT%20%2A%20WHERE%20TRUE```

and if unsubscribed:
```2023-01-26 10:48:11,281 INFO  io.mantisrx.api.push.MantisWebSocketFrameHandler [Salamander-ClientToZuulWorker-0] Channel [id: 0xb7e5e05b, L:/0:0:0:0:0:0:0:1:7004 ! R:/0:0:0:0:0:0:0:1:63289] is unregistered. URI: null```